components/Chips/tests/unit/ChipsTests.m

```diff
@@ -294,6 +294,27 @@ - (void)testPointInsideWithCustomHitAreaInsets {
                         withEvent:nil]);
 }
 
+- (void)testAddChipsManuallyPlaceholderCorrectPosition {
```

Comment:
Tip: Instead of checking the exact position of the placeholder text, consider testing that the addition of a Chip will increase the X-value of the placeholder's position.

Something like:

```
// When
CGFloat placeholderInitialXPosition = fakeField.placeholder.frame.origin.x;
[fakeField addChip:fakeChip];
[fakeField layoutIfNeeded];

// Then
CGFloat placeholderFinalXPosition = fakeField.placeholder.frame.origin.x;
XCTAssertGreaterThan(placeholderFinalXPosition, placeholderInitialXPosition);
```

What do you think?

Reply:
Done